sandbox/eo/app.eo

```diff
@@ -6,6 +6,5 @@
 [args...] > app
   stdout > @
     sprintf
-      "%dth Fibonacci number is %d\n"
-      (args.get 0).toInt > n
-      fibonacci n
+      "%f\n"
```

Comment:
@nlchar this change here is by mistake, right?

Reply:
@yegor256 sure. Fixed that. Thank you.